json-endpoints/jetpack/class.jetpack-json-api-themes-modify-endpoint.php

```diff
@@ -64,4 +64,48 @@ function update() {
 		return true;
 	}
 
+	function update_translations() {
```

Comment:
I'm assuming the translations and endpoint stuff shouldn't be a part of this PR, right?